airflow/jobs/scheduler_job.py

```diff
@@ -853,6 +853,32 @@ def process_file(
 
         ti_keys_to_schedule = self._process_dags(dags, session)
 
+        self._schedule_task_instances(dagbag, ti_keys_to_schedule, session)
+
+        # Record import errors into the ORM
+        try:
+            self.update_import_errors(session, dagbag)
+        except Exception:  # pylint: disable=broad-except
+            self.log.exception("Error logging import errors!")
+
+        return simple_dags, len(dagbag.import_errors)
+
+    @provide_session
+    def _schedule_task_instances(
+        self,
+        dagbag: models.DagBag,
```

Comment:
```suggestion
        dagbag: DagBag,
```
Can we use explicit type?

Reply:
In other places in this file we use this type of import. I would prefer to keep one convention. I prefer to limit additional changes if I make changes to the core. This makes the review more difficult.